sonatype-ci-release/src/main/scala/org/typelevel/sbt/TypelevelSonatypeCiReleasePlugin.scala

```diff
@@ -28,6 +29,10 @@ object TypelevelSonatypeCiReleasePlugin extends AutoPlugin {
       "Controls whether or not v-prefixed tags should be released from CI (default true)")
     lazy val tlCiReleaseBranches = settingKey[Seq[String]](
       "The branches in your repository to release from in CI on every push. Depending on your versioning scheme, they will be either snapshots or (hash) releases. Leave this empty if you only want CI releases for tags. (default: [])")
+    lazy val tlCiReleaseStepSummaryTableInfo = settingKey[Map[String, String]](
+      "Key-value set of information that will be rendered in a table in the job summary (default: [Release version -> ThisBuild/version])")
+    lazy val tlCiAddResolverInfoToSummary = settingKey[Boolean](
+      "If true adds the resolver url and instructions to the job summary (default: true)")
```

Comment:
On second thought—do we _really_ need to expose these as settings? Basically it constrains us since we'd have to support these settings going forward. E.g. if we decided a different, non-tabular way to display the release data.

Reply:
Makes sense. It pairs well with the combatibility question you raised in the global comment. I'll hide those settings as mere private vals.